demo/src/javascripts/components/demos/trips-layer/trips-layer.js

```diff
@@ -1,7 +1,8 @@
-import {Layer, assembleShaders} from '../../../../../../index';
 import {Model, Program, Geometry, glGetDebugInfo} from 'luma.gl';
+import {readFileSync} from 'fs';
+import {join} from 'path';
 
-const glslify = require('glslify');
+import {Layer, assembleShaders} from '../../../../../../index';
```

Comment:
Nit: Why are we moving this import line down? I think we should follow a logical ordering when including files (e.g. our own code first, then our own modules, then standard modules).